ngraph/core/src/op/abs.cpp

```diff
@@ -10,30 +10,27 @@
 #include "ngraph/runtime/host_tensor.hpp"
 #include "ngraph/runtime/reference/abs.hpp"
 
-using namespace std;
-using namespace ngraph;
+NGRAPH_RTTI_DEFINITION(ov::op::v0::Abs, "Abs", 0);
```

Comment:
```suggestion
OPENVINO_RTTI_DEFINITION(ov::op::v0::Abs, "Abs", 0, util::UnaryElementwiseArithmetic);
```

Reply:
Thanks, I will fix it in the separate PR.